pkg/services/services.go

```diff
@@ -102,8 +102,8 @@ func azureCloudNodeManagerConfiguration() servicescm.Service {
 
 // hybridOverlayConfiguration returns the Service definition for hybrid-overlay
 func hybridOverlayConfiguration(vxlanPort string, debug bool) servicescm.Service {
-	hybridOverlayServiceCmd := fmt.Sprintf("%s --node NODE_NAME --k8s-kubeconfig %s --windows-service "+
-		"--logfile "+"%s\\hybrid-overlay.log", windows.HybridOverlayPath, windows.KubeconfigPath,
+	hybridOverlayServiceCmd := fmt.Sprintf("%s --node NODE_NAME --bootstrap-kubeconfig=%s --cert-dir=%s --cert-duration=24h "+
```

Comment:
@kyrtapz  `--cert-duration=24h` means the cert will renew or expire in 24 h? 

Can the `hybrid overlay` service modify annotations after this duration?

Reply:
This means that the certificate will be valid for 24h, before it expires the hybrid-overlay binary will perform automatic rotation so there should be no issues there.